blueprints/lensappiq/README.md

```diff
@@ -9,7 +9,7 @@ This blueprint bootstraps a kind cluster and installs Lens AppIQ.
 ### Pre-req
 * [Install Boundless CLI `bctl`](https://github.com/Mirantis/boundless/blob/main/README.md#pre-requisite)
 
-### Bootsrap a Kind cluster with Lens AppIQ
```

Comment:
I would be nice to remove these since they have been copied to the content folder. That way we don't have to update two places. I'm not sure who all might be referring to this though

Reply:
Yes. We should actually remove this docs from Readme.md and provide a link for the website. Will create a separate ticket for it.

Reply:
I was just looking it over and going to suggest the same. I think we should provide links to the "getting started" and "examples" sections specifically in the readme